docs/developer/client.md

```diff
@@ -311,30 +275,21 @@ project's [TypeDoc][typedoc].
 
 [typedoc]: http://typedoc.org/ (TypeDoc)
 
-### src
+### lib
 
-The `src` directory contains the TypeScript source code.
+The `lib` directory contains the TypeScript source code.
 
 #### atomist.config.ts
 
-The `atomist.config.ts` file contains automation client
-project-specific configuration.
-See [Client Configuration][client-config] for more details.
-
-[client-config]: client.md#client-configuration (Atomist Automation Client Configuration)
-
-#### commands
-
-The `src/commands` directory contains the source code for commands.
-
-#### events
+The `atomist.config.ts` file contains 
+project-specific configuration. This is the starting point when you want to look at what this SDM might do.
+See [SDM Configuration][sdm-config] for more details.
 
-The `src/events` directory contains the source code for event
-handlers.
+[sdm-config]: client.md#sdm-configuration (Atomist SDM Configuration)
 
 #### typings
```

Comment:
If we have lib/graphql, this should probably be lib/typings and moved up after lib/graphql